cdk_framework/EKS/lib/config/clusters.yml

```
@@ -0,0 +1,17 @@
+---
+clusters:
+  armCluster:
+    launch_type: ec2
+    cpu_architecture: arm_64
+    version: 1.21
+    node_size: large
+  amdCluster0:
+    launch_type: ec2
+    cpu_architecture: AMD_64
+    version: 1.21
+    node_size: large
+  # fargateCluster0:
+  #   launch_type: fargate
+  #   cpu_architecture: null 
+  #   version: 1.22
+  #   node_size: large
```

```suggestion
  # fargateCluster0:
  #   launch_type: fargate
  #   cpu_architecture: null 
  #   version: 1.22
  #   node_size: large
```
remove dead code and instead add an example for a fargate definition